src/sas/qtgui/Calculators/media/sld_calculator_help.rst

```diff
@@ -2,57 +2,126 @@
 
 .. This is a port of the original SasView html help file to ReSTructured text
 .. by S King, ISIS, during SasView CodeCamp-III in Feb 2015.
+..
+    There is periodictable syntax for including density of components in the molecular formula field that does not appear to be implemented in SASview.
+..
+    For compounds, such as biomolecules, with exchangeable hydrogens, H[1] is used to denote the labile hydrogens. The reported contrast match point for the molecule takes into account the ratio of exchanged hydrogens.
+    This feature is not currently enabled in Sasview but is available on the NIST webpage.
 
 SLD Calculator Tool
 ===================
 
 Description
 -----------
+This tool calculates the neutron and x-ray scattering length densities (SLD) and some other useful scattering parameters of a wide range of materials including, molecules, solutions/mixtures, isotopic mixtures, and biomolecules.
+This SLD calculator utilizes the periodictable python package\ :sup:`1`, which is an extensible periodic table populated with values useful for neutron and x-ray experiments.
 
-The neutron scattering length density (SLD, $\beta_N$) is defined as
+User Inputs
+----------------------------
+**Molecular Formula** (required)
+    This field defines the material for which you are calculating the SLD. For more guidance on how to enter molecules, biomolecules, and more complex mixtures scroll down to "Specifying Materials or Mixtures in the Molecular Formula Field" section.
```

Comment:
Potential wordsmithing:
The section "Specifying Materials or Mixtures in the Molecular Formula Field" offers further guidance on how to format molecules, biomolecules, and more complex mixtures.

Reply:
Updated